src/polymer/polymer-element.ts

```diff
@@ -83,6 +83,7 @@ export class ScannedPolymerElement extends ScannedElement {
   //     level. Remove them here, they should only exist on PolymerElement.
   domModule?: dom5.Node;
   scriptElement?: dom5.Node;
+  slots: Slot[];
```

Comment:
Also the issue you're seeing with it being undefined is due it not giving it a default value. I wish the type system caught this, but apparently it does not.

So:

```typescript
slots: Slot[] = [];
```